rust/feature-flags/src/team/team_models.rs

```diff
@@ -15,15 +15,4 @@ pub struct Team {
     /// Thanks to this default-base approach, we avoid invalidating the whole cache needlessly.
     #[serde(default)]
     pub project_id: i64,
-    // TODO: the following fields are used for the `/decide` response,
-    // but they're not used for flags and they don't live in redis.
-    // At some point I'll need to differentiate between teams in Redis and teams
-    // with additional fields in Postgres, since the Postgres team is a superset of the fields
-    // we use for flags, anyway.
-    // pub surveys_opt_in: bool,
-    // pub heatmaps_opt_in: bool,
-    // pub capture_performance_opt_in: bool,
-    // pub autocapture_web_vitals_opt_in: bool,
-    // pub autocapture_opt_out: bool,
-    // pub autocapture_exceptions_opt_in: bool,
```

Comment:
RemoteConfig will handle decide, so I'm explicitly not going to parse any of this stuff from our teams table for flags.